web-app/django/VIM/apps/instruments/templates/instruments/index.html

```diff
@@ -196,4 +210,8 @@ <h4 class="ms-3 me-2 my-auto"><small>INSTRUMENT LIST</small></h4>
 <script src="https://unpkg.com/imagesloaded@5/imagesloaded.pkgd.min.js"></script>
 <script src="{% static "instruments/js/DisplaySettings.js" %}"></script>
 <script src="{% static "instruments/js/PaginationTools.js" %}"></script>
+<script>
```

Comment:
Can this go in one of our static `.js` files?

Reply:
Sure, I'll fix this today.